README.md

```diff
@@ -10,51 +11,131 @@ SketchyBar version.
 Feel free to contribute to this repo.
 
 ## LUA Module
+
 This code in this repository compiles into a lua module which can be
 required from any lua script.
 
 Install and update it with the command (this will build the lua module from
 source and place it in `$USER/.local/share/sketchybar_lua/`):
+
 ```bash
 (git clone https://github.com/FelixKratz/SbarLua.git /tmp/SbarLua && cd /tmp/SbarLua/ && make install && rm -rf /tmp/SbarLua/)
 ```
 
 For LUA to actually find the module, it has to reside in a path included in the
 lua cpath (TODO: Install module into the default lua cpath), e.g.:
+
 ```lua
 package.cpath = package.cpath .. ";/Users/" .. os.getenv("USER") .. "/.local/share/sketchybar_lua/?.so"
 ```
 
 The module can then be required in a lua script
+
 ```lua
 local sbar = require("sketchybar")
 ```
+
 and used to communicate with SketchyBar.
 
+## Using with [nix-darwin](https://github.com/LnL7/nix-darwin)
+
+You can install Stylix into your nix-darwin configuration via Flakes.
+Add this to your `flake.nix`:
+
+```nix
+# make sure you are using nixpkgs-unstable
+inputs.sbarlua = {
+    url = "github:FelixKratz/SbarLua";
+    nixpkgs.follows = "nixpkgs";
+};
+```
+
+Afterwards add this to your darwinConfiguration:
+
+```nix
+overlays = [
+    (final: prev: {
+    sbarlua = inputs.sbarlua.packages."${prev.system}";
+    })
+];
+```
+
+Finally add `pkgs.sbarlua.sbarlua` and `lua54Packages.lua` to your system packages like this:
+
+```nix
+environment.systemPackages = [
+    pkgs.sbarlua.sbarlua
+    pkgs.lua54Packages.lua # make sure to use this version of lua otherwise this won't work
+];
+```
+
+A minimal flake.nix containing SbarLua might look like this:
+
+```nix
+{
```

Comment:
In this block, besides changes to accommodate things mentioned elsewhere, there seems to be an excess of `inputs.` The only thing that should be named `inputs` is the block defining the inputs. The inputs themselves should not be prefixed, neither in the declaration nor later when calling them. If you intend to call the package, it is probably best to just add it as another parameter to the outputs function.